Sources/eppo/EppoClient.swift

```diff
@@ -45,14 +45,59 @@ public class EppoClient {
         self.flagConfigs = try JSONDecoder().decode(FlagConfigJSON.self, from: urlData);
     }
 
-    public func getAssignment(_ subjectKey: String, _ flagKey: String) throws -> String? {
+    public func getBoolAssignment(
+        _ subjectKey: String,
+        _ flagKey: String) throws -> Bool?
+    {
+        return try getAssignment(subjectKey, flagKey, SubjectAttributes())?.boolValue()
+    }
+    
+    public func getBoolAssignment(
+        _ subjectKey: String,
+        _ flagKey: String,
+        _ subjectAttributes: SubjectAttributes) throws -> Bool?
+    {
+        return try getAssignment(subjectKey, flagKey, subjectAttributes)?.boolValue()
+    }
+    
+    public func getNumericAssignment(
+        _ subjectKey: String,
+        _ flagKey: String) throws -> Double?
+    {
+        return try getAssignment(subjectKey, flagKey, SubjectAttributes())?.doubleValue()
+    }
+    
+    public func getNumericAssignment(
+        _ subjectKey: String,
+        _ flagKey: String,
+        _ subjectAttributes: SubjectAttributes) throws -> Double?
+    {
+        return try getAssignment(subjectKey, flagKey, subjectAttributes)?.doubleValue()
+    }
+    
+    public func getStringAssignment(
+        _ subjectKey: String,
+        _ flagKey: String) throws -> String?
+    {
+        return try getAssignment(subjectKey, flagKey, SubjectAttributes())?.stringValue()
+    }
+    
+    public func getStringAssignment(
+        _ subjectKey: String,
+        _ flagKey: String,
+        _ subjectAttributes: SubjectAttributes) throws -> String?
+    {
+        return try getAssignment(subjectKey, flagKey, subjectAttributes)?.stringValue()
+    }
+    
+    private func getAssignment(_ subjectKey: String, _ flagKey: String) throws -> EppoValue? {
         return try getAssignment(subjectKey, flagKey, [:]);
     }
 
-    public func getAssignment(
+    private func getAssignment(
```

Comment:
By making this private, could we be breaking things for SDK users not explicitly using the typed value methods yet?

Reply:
Yes this is a breaking change and we will set the version number to a "major" bump.